extensions/indexes/range/src/main/java/org/exist/indexing/range/RangeIndexConfigAttributeCondition.java

```diff
@@ -34,293 +34,217 @@
+        }
+
+        if (!elem.hasAttribute("attribute")) {
+            throw new DatabaseConfigurationException("Range index module: No attribute qname in condition");
         }
 
         attributeName = elem.getAttribute("attribute");
-        if (attributeName == null || attributeName.isEmpty()) {
-            throw new DatabaseConfigurationException("Range index module: Empty or no attribute qname in condition");
+        if (attributeName.isEmpty()) {
+            throw new DatabaseConfigurationException("Range index module: Attribute qname is empty");
         }
 
         try {
-            attribute = new QName(QName.extractLocalName(attributeName), XMLConstants.NULL_NS_URI,
-                    QName.extractPrefix(attributeName), ElementValue.ATTRIBUTE);
+            attribute = new QName(QName.extractLocalName(attributeName), XMLConstants.NULL_NS_URI, QName.extractPrefix(attributeName), ElementValue.ATTRIBUTE);
         } catch (final QName.IllegalQNameException e) {
-            throw new DatabaseConfigurationException("Rand index module error: " + e.getMessage(), e);
-        }
-        value = elem.getAttribute("value");
-
-        // parse operator (default to 'eq' if missing)
-        if (elem.hasAttribute("operator")) {
-            final String operatorName = elem.getAttribute("operator");
-            operator = Operator.getByName(operatorName.toLowerCase());
-            if (operator == null) {
-                throw new DatabaseConfigurationException(
-                        "Range index module: Invalid operator specified in range index condition: " + operatorName + ".");
-            }
-        } else {
-            operator = Operator.EQ;
+            throw new DatabaseConfigurationException("Range index module error: " + e.getMessage(), e);
         }
 
-        final String caseString = elem.getAttribute("case");
-        caseSensitive = (caseString != null && !caseString.equalsIgnoreCase("no"));
+        final String value = elem.getAttribute("value");
 
-        final String numericString = elem.getAttribute("numeric");
-        numericComparison = (numericString != null && numericString.equalsIgnoreCase("yes"));
-
-        // try to create a pattern matcher for a 'matches' condition
-        if (operator == Operator.MATCH) {
-            final int flags = caseSensitive ? 0 : CASE_INSENSITIVE;
-            try {
-                pattern = Pattern.compile(value, flags);
-            } catch (PatternSyntaxException e) {
-                RangeIndex.LOG.error(e);
-                throw new DatabaseConfigurationException(
-                        "Range index module: Invalid regular expression in condition: " + value);
-            }
-        }
+        operator = getOperator(elem);
+        commutative = (operator == Operator.EQ || operator == Operator.NE);
+        commutativeNegate = (operator == Operator.GT || operator == Operator.LT || operator == Operator.GE || operator == Operator.LE);
```

Comment:
`commutativeNegate` is a very strange name, at least in English. Perhaps `notCommutative` would be better to describe what you are trying to model?